app/helpers/application_helper.rb

```diff
@@ -185,7 +185,12 @@ def recap_item_list(holding)
   def recap_supervised_items?(holding)
     if holding.key? 'items'
       restricted_items = holding['items'].select { |item| item['use_statement'] == 'Supervised Use' }
-      !restricted_items.empty?
+      # !restricted_items.empty?
+      if restricted_items.count == holding['items'].count
+        true
+      else
+        false
+      end
```

Comment:
Lines 188-192 can be expressed with a single line:
```ruby
      restricted_items.count == holding['items'].count
```

Reply:
Thanks. Correct that.